### DISCUSSION TITLE
Discrepancies about size displayed on remote repo and size in Vorta.

### DISCUSSION THREAD

**Discussion by pardub** [borgbase/vorta#1704] [FAQ] [answered]
Hi,

I backup my laptop on 2 remote Borg repos with 2 different cloud providers.
On both of these Borg repos, I have around 117 GB of data backed up.
But on Vorta, the size shows of the folder backed up is  54.4 GB.
Also, on one of these remote repo, I can see my storage used increased by around 1 GB per day despite on Vorta , the daily archive  left on the this remote repo shows something  between 50 MB and 100 MB.

What could explain this difference ?

![remote_repo1](https://user-images.githubusercontent.com/42610272/234794139-22de4776-8d2e-4891-9734-49250a3587c8.png)
![Vorta_remote_repo1](https://user-images.githubusercontent.com/42610272/234794406-a790691a-bccd-4382-8056-b9a5bb9191ae.png)

![vorta size archives](https://user-images.githubusercontent.com/42610272/234794428-20b72689-fe9e-4c9b-9be3-0a641590dd56.png)

**Comment by m3nu** [comment]
One is the size of the source data, the other all different versions. Naturally they are different, especially if you have more than one archive. 😁

**Reply by pardub** [reply]
Thank you for the answer but I still didn’t get it.
My remote repo is pruned every day  and only one archive is left or each day.
For e.g,on the remote, we can can see the usage is 132.544 GB on the 20/04 and it was 131.047 GB on the day before, so 1.497 GB additional storage has been added on the remote  between the 19/04 and the 20/04 despite the only archive left that is shown on Vorta after pruning is 73.1 MB.

My assumption( obviously wrong) is that on the remote repo, my data usage should increased only by 73.1 MB and not by 1.497 GB?

![data_usage_19_04_2023](https://user-images.githubusercontent.com/42610272/234811532-5205ef84-2dd2-4267-97b4-9ad9479986b4.png)
![data_usage_20_04-2023](https://user-images.githubusercontent.com/42610272/234811555-50a003f4-d157-49f6-982c-8d24a9c2ba2d.png)

**Reply by real-yfprojects** [reply, marked as answer]
Do you regularly compact your repository? Borg will 'mark' archives as deleted but not actually free the disk space before running `borg compact`.

**Comment by pardub** [comment]
I don’t run any commands.
Vorta is starting automatically during the boot and the option “prune old Archive after backup” is always selected.
I never ran a `borg compact`.
Would you mind to let me know how to run this `borg compact` command in Vorta?

**Reply by pardub** [reply]
@real-yfprojects 
The command `borg compact` didn't seem to produce any effect from the shell commands area in Vorta.
But from the terminal, it did the job. I'm back on both remotes repo to something that matches the repo size displayed on Vorta. 
![hetzner_new_size](https://user-images.githubusercontent.com/42610272/234905301-0e4d2517-f0b2-4a6f-b232-e17158588d56.png)

Thank you very much for the help.
